holidayextras/ui-toolkit
# UXUI-284 Countdown Component

#### What does this PR do? (please provide any background)
- Creates a countdown component which allows a single date to be entered and counts down the seconds, minutes, hours and days between then and the current time.

#### What tests does this PR have?
- All tests can be found in the _tests_ folder of the component
- To run the tests use `npm test`

#### How can be this tested?
- To manually test, run `grunt docs` from a local copy.
- The docs will start in your browser, scroll down to "countdown" and you will see a working version. You can edit the `code` on the docs and change the date to whatever you see fit.

#### Screenshots / Screencast
<img width="862" alt="screen shot 2015-08-03 at 12 13 04" src="https://cloud.githubusercontent.com/assets/778942/9036382/03b9367c-39d9-11e5-8262-4dba2aa14485.png">


#### What gif best describes how you feel about this work?
![Countdown](http://ak-hdl.buzzfed.com/static/2015-02/6/12/enhanced/webdr11/anigif_enhanced-11081-1423242139-14.gif)


---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
